chain/chain/src/stateless_validation/chunk_validation.rs

```diff
@@ -130,16 +130,30 @@ pub fn pre_validate_chunk_state_witness(
     // Blocks from the last last new chunk (exclusive) to the last new chunk (inclusive).
     let mut blocks_after_last_last_chunk = Vec::new();
 
-    {
+    let (last_chunk_shard_id, last_chunk_shard_index) = {
```

Comment:
nit: Can this be moved to a helper method?

Reply:
Done!